### ISSUE TITLE
Update case name command has a misleading description 

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2771]
This is the syntax to update a case:
```
Usage: scout update case [OPTIONS] [CASE_ID]

  Update a case in the database

Options:
  -n, --case-name TEXT           Add/update the display name of case
  -i, --institute TEXT           Update what institutes that has access to
                                 case
  -c, --collaborator TEXT        Add a collaborator to the case
  --vcf PATH                     path to clinical VCF file to be added
  --vcf-sv PATH                  path to clinical SV VCF file to be added
  --vcf-cancer PATH              path to clinical cancer VCF file to be added
  --vcf-cancer-sv PATH           path to clinical cancer structural VCF file
                                 to be added
  --vcf-research PATH            path to research VCF file to be added
  --vcf-sv-research PATH         path to research VCF with SV variants to be
                                 added
  --vcf-cancer-research PATH     path to research VCF with cancer variants to
                                 be added
  --vcf-cancer-sv-research PATH  path to research VCF with cancer structural
                                 variants to be added
  --reupload-sv                  Remove all SVs and re upload from existing
                                 files
  --rankscore-treshold TEXT      Set a new rank score treshold if desired
  --rankmodel-version TEXT       Update the rank model version
  --sv-rankmodel-version TEXT    Update the SV rank model version
  --help                         Show this message and exit.
```

According to the command's description, `-n, --case-name` makes you update a case name, but this is actually incorrect. Case name and institute are instead used to understand which case should be updated whenever a case ID is not provided with the command. Relative code is here: https://github.com/Clinical-Genomics/scout/blob/6171fff1f4b68b03f690b9155fcae16abe76b5e8/scout/commands/update/case.py#L86

**Comment by mikaell** [comment]
I've played around a bit with the update command. But I can't really make sense of the `--case-name` command. 

With intention to set vcf for the case where display_name is 643594 and institute is cust000:

```
$ scout update case --vcf  /Users/mikael/asdf -i cust000 -n 643594
2021-08-20 15:52:58 mikaellmbp scout.commands.base[48513] INFO Running scout version 4.37
2021-08-20 15:52:58 mikaellmbp scout.commands.base[48513] DEBUG Debug logging enabled.
2021-08-20 15:52:58 mikaellmbp scout.commands.update.case[48513] DEBUG *****
2021-08-20 15:52:58 mikaellmbp scout.commands.update.case[48513] DEBUG case_id: None
2021-08-20 15:52:58 mikaellmbp scout.commands.update.case[48513] DEBUG case_name: 643594
2021-08-20 15:52:58 mikaellmbp scout.commands.update.case[48513] DEBUG institute: cust000
2021-08-20 15:52:58 mikaellmbp scout.commands.update.case[48513] DEBUG *****
2021-08-20 15:52:58 mikaellmbp scout.adapter.mongo.case[48513] INFO Fetching case cust000-643594
2021-08-20 15:52:58 mikaellmbp scout.commands.update.case[48513] WARNING Case cust000-643594 could not be found
Aborted!
```

Looking into the code, to me it seems like the institute and display_name are concatenated and used as argument for a MongoDB lookup, which fails.  Is my understanding correct?


```
def case(
    context,
    case_id,
    case_name,
    institute,
    collaborator,
    vcf,
    vcf_sv,
    vcf_cancer,
    vcf_cancer_sv,
    vcf_research,
    vcf_sv_research,
    vcf_cancer_research,
    vcf_cancer_sv_research,
    reupload_sv,
    rankscore_treshold,
    rankmodel_version,
    sv_rankmodel_version,
):
    """
    Update a case in the database
    """
    
    adapter = store
    if not case_id:
        if not (case_name and institute):
            LOG.info("Please specify which case to update.")
            raise click.Abort()
        case_id = "{0}-{1}".format(institute, case_name)
    # Check if the case exists
    case_obj = adapter.case(case_id)

    if not case_obj:
        LOG.warning("Case %s could not be found", case_id)
        context.abort()
```

**Comment by northwestwitch** [comment]
That's, correct. This will be fixed when we merge #2727. I think that concatenation institute_case-name is some defunct way of naming cases IDs in the past.
